regression-tests/run-tests.sh

```diff
@@ -195,7 +195,7 @@ for test_file in $tests; do
         # The source is temporarily copied to avoid issues with bash paths in cl.exe
         (cd $exec_out_dir; \
          cp ../../$expected_src $generated_cpp_name;
-         $compiler_cmd "$test_bin" \
+         $compiler_cmd"$test_bin" \
```

Comment:
So sticking those together was necessary in the end? I think it is the cl.exe argument format that requires that, isn't it?
Please maybe add a comment about that.

Reply:
I don't know.
I just reverted the change.

Reply:
OK, it is not worth extra effort.